doc/tools/apigen.py

```diff
@@ -207,7 +207,11 @@ def _parse_module_with_import(self, uri):
         classes : list of str
             A list of (public) class names in the module.
         """
-        mod = __import__(uri, fromlist=[uri])
+        try:
+            mod = __import__(uri, fromlist=[uri])
+        except ImportError:
+            # fix ImportError for submodules in Python 3
+            mod = __import__(uri, fromlist=[])
```

Comment:
I think this will only import the base package module, so `nibabel` when `uri='nibabel.parrec'`.  At least for me, you need `fromlist=['']` to get the submodule, which I think is what we want.  That seems to work for Python 2 and Python 3.